app/models/user.rb

```diff
@@ -1,11 +1,8 @@
 class User < ApplicationRecord
   belongs_to :role
   has_many :inventories
-
-  has_secure_password
-
+  after_initialize :set_default_role, if: :new_record?
```

Comment:
`before_create` ?

Reply:
I've just added this line. before_create already exists o develop branch

Reply:
I must create SuperAdmin with seeding. And it seemed that I will overwrite his role_id using before_create. Because of that I decided to use after_initialize

Reply:
Try `||=`